lib/vintage_net_wifi.ex

```diff
@@ -995,4 +995,47 @@ defmodule VintageNetWiFi do
   end
 
   defp trim_orphan_backslash(s), do: s
+
+  @doc """
+  Experimental API for getting WiFi driver capabilities
+
+  This queries the `wpa_supplicant` and driver to see what it supports. It's
+  useful for seeing whether WPA3, 5 GHz, and other things are supported. The
+  results aren't currently processed.
```

Comment:
```suggestion
  This queries the `wpa_supplicant` and driver to see what it supports. It's
  useful for seeing whether WPA3, 5 GHz, and other things are supported. The
  results aren't currently processed.

  For any attributes not in the default list, use `GET_CAPABILITY` with `WPASupplicant.raw_command/2`
```

The list of keys for capabilities looks complete enough, so just an idea for a helpful sentence in the doc in case someone is looking for something not included. Thoughts?

Reply:
I grabbed the capabilities from reading the source code. The main reason that I marked this as experimental is that aside from a few capabilities, I think you really need to dig into the source to know what they mean. Even then, the TI Wilink8 driver advertises WPA3 support and is still missing some feature that prevents it from working (reason "unknown"... grrr...)